src/services/FulfillmentService.js

```diff
@@ -99,6 +100,18 @@ class FulfillmentService {
       callback(err);
     }
   };
+
+  saveMessage = async req => {
+    try {
+      const chatbotHistory = new ChatbotHistory({ data: req.body });
+      chatbotHistory.save();
+
+      return 'Thank you for your input. It has been logged.';
```

Comment:
this return needs to be in the reponse data structure's shape that is documented in the story.

Reply:
done